### ISSUE TITLE
[Code Addition Request]: Alpaca_Identification

### ISSUE THREAD

**Issue by deepanshubaghel** [UTSAVS26/PyVerse#491]
### Have you completed your first issue?

- [X] I have completed my first issue

### Guidelines

- [X] I have read the guidelines
- [X] I have the link to my latest merged PR

### Latest Merged PR Link

#426 

### Abstract
This project is all about image classification by using the
deep neural network (DNN) or also known as Deep Learning
by using framework TensorFlow. Python is used as a
programming language because it comes together with
TensorFlow framework. The input data mainly focuses in
flowers category which there are five (5) types of flowers that
have been used in this paper. Deep neural network (DNN) has
been choosing as the best option for the training process
because it produced a high percentage of accuracy. Results are
discussed in terms of the accuracy of the image classification
in percentage. Here we are going to use Alpaca Images for creating the classification model which will identify the images of Alpaca from the user input.

### Dataset
The dataset which is used here, is collected from Kaggle website. Here is the link of the dataset : https://www.kaggle.com/sid4sal/alpaca-dataset-small.

### Goal
The goal of this project is to make a identification cum classification model using deep neural networks, which will identify the images of Alpaca from the user given input.
******************************************

### Full Name

Deepanshu Baghel

### Participant Role

gssoc-ext

**Comment by deepanshubaghel** [comment]
@UTSAVS26
